src/VehicleSetup/JoystickConfigController.h

```diff
@@ -220,7 +220,7 @@ private slots:
     static const int _attitudeControls = 5;
     
     int _axisCount;                     ///< Number of actual joystick axes available
-    static const int _axisMax = 4;      ///< Maximum number of supported joystick axes
+    static const int _axisMax = 6;      ///< Maximum number of supported joystick axes
```

Comment:
Same issue here: If the user has yaw on axis 8 he still will run into the same issue.